src/fides/api/ops/schemas/redis_cache.py

```diff
@@ -23,8 +23,3 @@ class Identity(IdentityBase):
 
     ga_client_id: Optional[str] = None
     ljt_readerID: Optional[str] = None
-
-    class Config:
-        """Only allow phone_number, email, and GA client id to be supplied"""
-
-        extra = Extra.forbid
```

Comment:
Is this an intentional change we're making right now @seanpreston? Seems like this should be separate.

Reply:
Have updated this. In this iteration it the `Config` was inherited from the `IdentityBase` class so it wasn't actually removed.

Reply:
thank you for clarifying 👍